types/redux-action/redux-action-tests.ts

```diff
@@ -1,5 +1,4 @@
 import * as ReduxAction from 'redux-action';
-import * as ReduxThunk from 'redux-thunk';
```

Comment:
Why not re-import the new `redux-thunk` types from DefinitelyTyped? I'd imagine that would fix the build errors.

Reply:
They don't appear to be used at all. When you git blame this line, it was added in isolation, without any other code changes to depend on the ReduxThunk types.

Reply:
Gotcha. I had assumed maybe there was some global overloading that `redux-thunk` did to `dispatch`. But you're right, that was a strange commit :)